src/utils/helpers.py

```diff
@@ -240,11 +239,14 @@ def get_path_of_ldk(ldk_data_name: str) -> str:
         CommonException: If an error occurs while accessing or creating the folder.
     """
     try:
+        current_network = NetworkEnumModel(bitcoin_network.__network__)
         local_storage_base_path = local_store.get_path()
         if not local_storage_base_path:
             raise CommonException('Unable to get base path of application')
 
-        data_ldk_path = os.path.join(local_storage_base_path, ldk_data_name)
+        data_ldk_path = os.path.join(
+            local_storage_base_path, current_network, ldk_data_name,
+        )
```

Comment:
I see the definition of the app's directory in many places, please deduplicate this. The app at startup should fill some runtime constants with the paths it frequently uses